routes.go

```diff
@@ -28,7 +28,8 @@ const (
 	JobBadge       = "JobBadge"
 	MainJobBadge   = "MainJobBadge"
 
-	ClearTaskCache = "ClearTaskCache"
+	ClearTaskCache          = "ClearTaskCache"
+	ClearTaskCacheQueryPath = "CachePath"
```

Comment:
This should be `"cache_path"` since that's how we format all the other query params.